frontend/src/store.js

```diff
@@ -76,17 +76,13 @@ export default new Vuex.Store({
     ...screeningActions,
     ...matchingActions,
     ...reasonActions,
-    ...matchesActions
+    ...matchesActions,
   },
   getters: {
     ...studentGetters,
     ...nativeLanguageGetters,
     ...teacherGetters,
     ...reasonGetters,
     ...matchesGetters,
-    screeningStudents: (state) => state.students.filter((student) => student.status.Description === "SCREENING"),
-    unmatchedStudents: (state) => state.students.filter((student) => student.status.Description === "UNMATCHED"),
-    getStudentByStudentId: (state) => (id) => state.students.find(student => student.StudentID == id),
-    getTeacherByTeacherId: (state) => (id) => state.teachers.find(teacher => teacher.TeacherID == id)
```

Comment:
Is this already in the Teachers store file? Because it's not added in this PR. On my phone right now so can't check.

Reply:
Yeah it is. I just checked.